src/utils/components/DiskModal/components/StorageProfileSettings/RecommendationLabel.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Label, Tooltip } from '@patternfly/react-core';
+
+type RecommendationLabelProps = {
+  priority: number;
+  recommendationCount: number;
+};
+
+const tooltipForOtherRecommended = (priority: number, t) => {
+  switch (priority) {
+    case 1:
+      return t('The 2nd most suggested option in StorageProfile.');
+    case 2:
+      return t('The 3rd most suggested option in StorageProfile.');
+    default:
+      return t('The {{priority}}th most suggested option in StorageProfile.', {
+        priority: priority + 1,
+      });
+  }
+};
+
+const fromPriorityToLabels = (priority, recommendationCount, t) => {
```

Comment:
this need to be in a util file and no need to send t as arg

Reply:
please add types to function

Reply:
> no need to send t as arg
>
It depends if we want to support changing the language without reloading the page. If not then we can move the translations outside of React lifecycle.